src/util/vocabularyChecker/detectWrongSyntax.test.ts

```diff
@@ -134,7 +134,7 @@ describe("CollectAllInvalidWordsInIssues", () => {
 				message: "Das Wort 'Sonne' in der Eingabe ist nicht erlaubt.",
 				position: {
 					fromIndex: 0,
-					toIndex: 5,
+					toIndex: 4,
```

Comment:
I think this is not correct. I shouldn't return index 4 as until. This would only include "Sonn" then

Reply:
I think 0 to 4 should be right, as those are exactly 5 letters. That's why I changed it to -1 later, too. I can change it back again, tho, if you insist.